tests/Resources/ti.ui.shortcutitem.test.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+
+const should = require('./utilities/assertions');
+
+describe('Titanium.UI.ShortcutItem', () => {
+
+	it.android('Ti.UI.ShortcutItem', () => {
+		should(Ti.UI.ShortcutItem).not.be.undefined();
+	});
+
+	it.android('createShortcutItem', () => {
+		should(Ti.UI.createShortcutItem).not.be.undefined();
+		should(Ti.UI.createShortcutItem).be.a.Function();
+
+		// ONLY compatible with Android 7.1+, end test early.
+		if (Ti.Platform.osname === 'android') {
+			const version = Ti.Platform.version.split('.');
+			if (parseInt(`${version[0]}${version[1]}`) < 71) {
+				return;
+			}
+		}
+
+		// Create shortcut item.
+		const item = Ti.UI.createShortcutItem({
```

Comment:
We often only test the "golden path" and that's a good first start. May also want to consider - what if we don't supply some of these properties? Are some required? If so, what do we expect it to do then, throw an Error? Do we expect them to adopt default values? What if we pass in values with incorrect types - do we expect an Error? implicit conversion?